list/List.js

```diff
@@ -564,10 +564,10 @@ define([
 		 * @protected
 		 */
 		handleSelection: function (/*Event*/event) {
-			var eventRenderer = this.getEnclosingRenderer(event.target);
-			if (eventRenderer) {
-				if (!this.isCategoryRenderer(eventRenderer)) {
-					this.selectFromEvent(event, eventRenderer.item, eventRenderer, true);
+			var renderer = this.getEnclosingRenderer(this.navigatedDescendant);
```

Comment:
Should just be able to use `this.navigatedDescendant` rather than `this.getEnclosingRenderer(this.navigatedDescendant)`, but that doesn't explain the test failures.